src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -17,7 +19,7 @@ namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
     [Authorize(Policy = PolicyName.CanAccessOrganisations)]
     [Route("api/v1/Organisations")]
     [ApiController]
-    [Produces("application/json")]
+    [Produces(MediaTypeNames.Application.Json)]
     public sealed class OrganisationsController : Controller
```

Comment:
We should update this to derive from `ControllerBase` while we're here.

Reply:
👍